### ISSUE TITLE
Token request: Adding Fuse network token

### ISSUE THREAD

**Issue by bejavu** [Uniswap/uniswap-frontend#493]
**Please provide the following information for your token.**

Token Address: 0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d
Token Name (from contract): Fuse Token
Token Decimals (from contract): 18
Token Symbol (from contract): FUSE
Uniswap Exchange Address of Token: 0xf9a204774ac595216c3c4e4249b7624e34ee52d5

Link to the official homepage of token: https://fuse.io
Link to CoinMarketCap page of token: -

Some tokens (e.g. BNB) do not work with Uniswap v1. In order to assess if your token works correctly, please complete small-value transactions of each of the types below, and submit the Etherscan transaction links for our review.
Test `addLiquidity` transaction: https://etherscan.io/tx/0xe18b83df917b4c937f799febcfaea0836cb222ceab72d947444403da1dcb992b
Test `swap` transaction: https://etherscan.io/tx/0x9d46c87d1e33e50d4124ad6e78b7beb90361fd7620a9c2f96734b6c66d415897
Test `removeLiquidity` transaction: https://etherscan.io/tx/0x698e194cc89024d4f77437a2c4e2ea08b4bc144d5892ec9d3cbf6f6afc80c109

Are you willing to add liquidity to the liquidity pool for this token? (Y/N): Y
If so, how much liquidity are you willing to add?: up to 100 ETH

**Comment by ashleigh-schap** [comment]
@bejavu Has this project launched? The block explorer says there are only 3 holders of this token which is not enough to warrant adding it to the frontend of uniswap.

**Comment by bejavu** [comment]
> @bejavu Has this project launched? The block explorer says there are only 3 holders of this token which is not enough to warrant adding it to the frontend of uniswap.

The project had launched already, and most of the traffic and holders are on the Fuse Chain (a side chain to ethereum - https://explorer.fuse.io/)

We want to grow liquidity on uniswap and already have traction (you can see in the explorer) and need a little bit of help from you to get started.

**Comment by CrackerHax** [comment]
More than 3 mainnet holders: 
https://ethplorer.io/address/0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d
 
And on fusenet:
https://explorer.fuse.io/accounts

**Comment by fileflume** [comment]
More transactions happening now as the project develops and companies are using the newwork. Would be great to have Fuse as an official Uniswap token.

**Comment by Andrew-Pohl** [comment]
Yeah get Fuse on uniswap. Great team, active community with increasing network interest.

**Comment by Marksmargon** [comment]
Yes - this project has grown since - please add Fuse to your UI!
